rspirv/sr/autogen_ops.rs

```diff
@@ -18,9 +18,31 @@
 
 use crate::sr::{Token, Type};
 #[derive(Clone, Debug, Eq, PartialEq)]
+pub enum Branch {
+    Branch {
+        target_label: spirv::Word,
+    },
+    BranchConditional {
+        condition: spirv::Word,
```

Comment:
Here `condition` should be `Token<Instruction>` and labels should be `Token<BasicBlock>`.

If you intend to change it later, would be nice at least to have some TODO in the source code. The existing implementation seems wrong to me.